cmd/server/assets/users.html

```diff
@@ -88,6 +95,15 @@ <h2>Users</h2>
                 </form>
               {{end}}
             </td>
+            <td scope="row">
+              {{index $codesGenerated1d .ID}}
```

Comment:
Is there a reason you can't do `$codesGenerated1d.ID`?

Reply:
This is the id of a user being used as an index into the map.